R/pkg/tests/fulltests/test_sparkSQL.R

```diff
@@ -1502,12 +1502,21 @@ test_that("column functions", {
   result <- collect(select(df, sort_array(df[[1]])))[[1]]
   expect_equal(result, list(list(1L, 2L, 3L), list(4L, 5L, 6L)))
 
-  # Test flattern
+  result <- collect(select(df, reverse(df[[1]])))[[1]]
```

Comment:
Seems we don't have test for `reverse` for string. Can you add one for it too?

Reply:
@viirya Thank for your comments. I will make changes.